front/src/styles/scss/applications/operationalStudies/_pacedTrain.scss

```
@@ -121,8 +131,134 @@
   }
   .occurrences {
     width: 100%;
-    //TODO PACED TRAIN: set the height to auto when integrating the OccurenceItem component
-    height: 20px;
+    height: auto;
+    padding-left: 28px;
+    margin-top: 12px;
+
+    .occurrence-item {
+      position: relative;
+      display: flex;
+      width: 100%;
+      flex-wrap: nowrap;
+      padding-left: 4px;
+      height: 32px;
+      align-items: center;
+      border-top: 1px solid var(--white100);
+      border-bottom: 1px solid var(--black10);
+      &.selected {
+        background-color: var(--selection20);
+      }
+      &.after-midnight {
+        border-bottom: 1px solid var(--black50);
+      }
+      &-name {
+        white-space: nowrap;
+        overflow: hidden;
+        /* "overflow" value must be different from "visible" */
+        text-overflow: ellipsis;
+        width: 56px;
+        direction: rtl;
+        text-align: left;
+        height: 24px;
+        line-height: 24px;
+        min-width: 56px;
+        margin-inline: 8px;
+      }
+
+      &:last-child {
+        border-bottom: none;
+      }
+      &:first-child {
+        border-top: none;
+      }
+      .occurrence-item-dot {
+        height: 24px;
+        color: var(--grey40);
+      }
+      .status-icon {
+        margin-left: 8px;
+        min-width: 16px;
+        height: 24px;
+      }
+
+      .rolling-stock {
+        display: flex;
+        align-items: center;
+        overflow: hidden;
+        height: 24px;
+        position: relative;
+
+        .status-invalid {
+          width: 4px;
+          height: 16px;
+          background-color: var(--error60);
+          border-radius: 5px;
+          margin: 0 7px 0 5px;
+        }
+        &::after {
+          content: '';
+          position: absolute;
+          top: 0;
+          left: 0;
+          width: 24px;
+          height: 22px;
+          background: linear-gradient(270deg, rgba(247, 246, 238, 0), rgba(247, 246, 238, 1));
+          z-index: 1;
+        }
+
+        img {
+          min-width: 40px;
+          height: 16px;
+          object-fit: cover;
+          object-position: left;
+          transform: scaleX(-1);
+        }
+      }
+
+      &-times {
+        display: flex;
+        margin-left: 8px;
+        margin-right: 16px;
+      }
+
+      &-time {
+        width: 44px;
+        text-align: right;
+        &.arrival-time {
+          margin-left: 32px;
+        }
+      }
+      .consecutive-day-date-display {
```

```suggestion
      .consecutive-day-timeline-display {
```

just because it's a bit weird to have `date-display` nested, it could be a source of error during future development

---

There is no day's timeline here just date of the days and number of month maybe I don't get your comment

---

But i don't have passion so let's go

---

hum ok it's fine